skyxplore-api-test/src/test/java/com/mentoring/test/CharacterTest.java

```diff
@@ -0,0 +1,40 @@
+package com.mentoring.test;
+
+import com.mentoring.endpoints.CharacterEndpoint;
+import com.mentoring.framework.BaseApiTest;
+import com.mentoring.generator.User;
+import com.mentoring.utilities.UserUtils;
+import org.assertj.core.api.Assertions;
+import org.testng.annotations.Test;
+
+public class CharacterTest extends BaseApiTest {
+
+    @Test(groups = SMOKE)
+    public void characterCrud() {
+        User user = new User();
+
+        CharacterEndpoint characterEndpoint = new CharacterEndpoint();
+        characterEndpoint.createCharacter(UserUtils.generateRandomCharacterName(), user.getAuthCookie(), 200);
+        characterEndpoint.getAllCharacters(user.getAuthCookie(), 200);
+
+        String characterId = characterEndpoint.getCharacterResponse().getCharacterId();
+        Assertions.assertThat(characterEndpoint.getAllCharactersResponse())
+                .as("All characters list should contain created character response.")
+                .contains(characterEndpoint.getCharacterResponse());
+
+        String updated_name = UserUtils.generateRandomCharacterName();
```

Comment:
Please use camelCase naming instead of snake_case

Reply:
Seems like Python has grown on me :D